nidirect_gp/nidirect_gp.module

```diff
@@ -130,6 +149,33 @@ function nidirect_gp_entity_presave(EntityInterface $entity) {
   $entity->setTitle($title);
 }
 
+/**
+ * Implements hook_views_pre_render().
+ */
+function nidirect_gp_views_pre_render(ViewExecutable $view) {
+
+  if ($view->id() == 'gp_practices' && $view->current_display == 'find_a_gp') {
+    // Display singular or plural result count.
+    $result_count = Drupal::translation()->formatPlural(
```

Comment:
Should this be `\Drupal::translation()` (leading backslash)?

Reply:
Does work without but for standards I'll add it in.